schemas/cloudToDevice/pgnss/pgnss-config-example.json

```diff
@@ -0,0 +1,4 @@
+[
+    "pgps.nrfcloud.com",
```

Comment:
Can we add add a DNS CNAME record for pgnss.nrfcloud.com?

Reply:
Done here: https://github.com/nRFCloud/backend/pull/2249